app/models/grda_warehouse/tasks/identify_duplicates.rb

```diff
@@ -44,55 +45,67 @@ def identify_duplicates
       # If we do find a match:
       #   create the associated WarehouseClient
 
-      Rails.logger.info "Matching #{unprocessed.count} unprocessed clients"
+      unprocessed_count = unprocessed.count
+      Rails.logger.info 'Matching #unprocessed_count} unprocessed clients'
```

Comment:
wrong quote type for interpolation
```suggestion
      Rails.logger.info "Matching #unprocessed_count} unprocessed clients"
```